IntegrationTests/Fixtures/XCBuild/Libraries/Bar/Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
     name: "Bar",
     products: [
-        .library(name: "BarLibProduct", type: .dynamic, targets: ["BarLib"]),
+        .library(name: "BarLib", type: .dynamic, targets: ["BarLib"]),
```

Comment:
Will packages require this kind of change in order to work with this PR?  I am pretty sure that targets and products are allowed to have the same name, it's just that two targets or two products cannot have the same name.

Reply:
Nope, I'm just reverting #3686 so that we test again that the names can be the same.

Reply:
I see — I didn't realize this was a revert PR.